src/components/Button.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './Button.css';
 
-function Button() {
+function Button({ buttonName }) {
   return (
     <div className="Button" id="button">
-      <p> Calculator App: Hello world!</p>
+      <p>
+        { buttonName}
+      </p>
     </div>
   );
 }
 
+Button.defaultProps = {
+  buttonName: 'defaultName',
+};
+
+Button.propTypes = {
+  buttonName: PropTypes.string,
+};
```

Comment:
- [ ] Let's declare the `buttonName` prop as a required prop. We can declare this way instead.
```JS
Button.propTypes = {
  buttonName: PropTypes.string.isRequired,
};
```